cmd/internal/flowmode/cluster_builder_test.go

```diff
@@ -0,0 +1,18 @@
+package flowmode
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuildClusterService(t *testing.T) {
+	opts := clusterOptions{
+		JoinPeers:     []string{"foo", "bar"},
```

Comment:
Shouldn't we have a test with `JoinPeers` empty string somewhere, or setting a flag to it?

Reply:
Yes, pushing it.

Reply:
(sorry, didn't post the comment beforehand).

After taking a look we don't have a good place for putting tests related to flag parsing. I'd like to (sooner or later) move ahead with #4973 which would cover this. I'm merging this to move ahead with the patch release.